src/modules/client/pages/auxiliaries/customers/AuxiliaryCustomersDirectory.vue

```diff
@@ -80,9 +81,6 @@ export default {
         console.error(e);
       }
     },
-    goToCustomerProfile (row) {
```

Comment:
que penses tu de laisser la gestion du router ici
tu pourrais dans le composant TableList emettre un evenement `go-to` et relier l'evenement a cette methode ; ca me parait peut etre plus simple dans le code et plus flexible dans ce qu'on pourrait faire en redirection

Reply:
vu ensemble, ca ne fonctionne pas